public/scripts/browse.js

```diff
@@ -40,7 +40,24 @@
 
     addSearchBar(myMap);
 
-    addMarkers(myMap);
+    if (document.getElementById('myMap').dataset.origin === "browse") {
+      addMarkers(myMap);
+    }
+
+    if (document.getElementById('myMap').dataset.origin === "report") {
+      var marker;
```

Comment:
might be neater to separate this out into its own function (lines 48 to 59) - similar to how `addSearchBar` and `addMarkers` are separated out